### PR TITLE
Remove React imports 

### PR DESCRIPTION
**Pull request by Tristan-WorkGH**
Since React 17, it's [no longer necessary to `import React from 'react'`](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).

[_goodbye-react-imports-17_](https://www.linkedin.com/pulse/say-goodbye-react-imports-17s-game-changing-feature-deepak-sharma-)

### CODE REVIEW

**Review by flomillot — APPROVED** [review]

File: src/components/ElementSearchDialog/element-search-dialog.jsx [hunk 1]
```diff
@@ -133,14 +133,14 @@ const ElementSearchDialog = (props) => {
                             InputProps={{
                                 ...params.InputProps,
                                 startAdornment: (
-                                    <React.Fragment>
+                                    <Fragment>
```
Comment (sBouzols) [inline src/components/ElementSearchDialog/element-search-dialog.jsx:136]: <></> instead of Fragment ?

**Review by sBouzols — CHANGES_REQUESTED** [review]
Prefer `<></>` usage instead of importing `Fragment`

**Review by sBouzols — APPROVED** [review]

**Comment by Tristan-WorkGH** [comment]
strange, the CI have run, but the status isn't reported here...  
[Tristan-WorkGH/rte-gridsuite-commons-ui/actions](https://github.com/Tristan-WorkGH/rte-gridsuite-commons-ui/actions/workflows/build.yml)

**Comment by Tristan-WorkGH** [comment]
Asked for help on https://github.com/orgs/community/discussions/113526

**Pull request merged**